app/src/main/java/uk/ac/ebi/atlas/experimentpage/cellplot/CellPlotType.java

```diff
@@ -0,0 +1,19 @@
+package uk.ac.ebi.atlas.experimentpage.cellplot;
+
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.ImmutableSet;
+
+import java.util.Optional;
+
+public class CellPlotType {
+    private final static ImmutableMap<String, ImmutableSet<String>> PLOT_OPTIONS =
+            ImmutableMap.of(
+                    "tsne", ImmutableSet.of("perplexity"),
+                    "umap", ImmutableSet.of("n_neighbors"));
+
+    public final static String DEFAULT_PLOT_METHOD = "umap";
```

Comment:
A minor comment, but wouldn't it be better to name it as `DEFAULT_CELL_PLOT_TYPE` just to make it consistent with the class name.

Reply:
I chose that name to align it with the request parameter `plotMethod` in the class `JsonCellPlotController`. I modelled a plot type as two values: a plot mehotd and a set of parameters. E.g. a UMAP plot type has a plot method `umap` and a set of parameters with a single element, `n_neighbors`. Put it another way, a plot method is the identifier of a plot type.